elements/lisk-api-client/src/ws_channel.ts

```diff
@@ -130,14 +149,9 @@ export class WSChannel {
 			params: params ?? {},
 		};
 
-		const send = new Promise((resolve, reject) => {
-			this._ws?.send(JSON.stringify(request), (err): void => {
-				if (err) {
-					return reject(err);
-				}
-
-				return resolve();
-			});
+		const send = new Promise(resolve => {
+			this._ws?.send(JSON.stringify(request));
+			resolve();
```

Comment:
not strictly related to this PR, but the send doesn't need to be wrapped with promise.
`ACKNOWLEDGMENT_TIMEOUT` should be used in place of `RESPONSE_TIMEOUT`